File: strobe_wiring.py
import wiringpi

# wiringpi.wiringPiSetupGpio
# wiringpi.pinMode(4, wiringpi.OUTPUT) # set pin 7 as output

# while True:
	# wiringpi.digitalWrite(7, 1)
	# wiringpi.digitalWrite(7, 0)

io = wiringpi.GPIO(wiringpi.GPIO.WPI_MODE_PINS)

io.pinMode(7,io.OUTPUT)

while True:
	io.digitalWrite(7,io.HIGH)
	io.digitalWrite(7,io.LOW)

File: strobe_wiring.c
#include <wiringPi.h>

#include <stdio.h>
#include <stdlib.h>
#include <stdint.h>

int main() {
  if (wiringPiSetup () == -1)
    exit (1) ;

  pinMode(7, OUTPUT);

  while(1) {
    digitalWrite(7, 0);
    digitalWrite(7, 1);
  }

  return 0 ;
}
